languages/en.ini

```diff
@@ -240,6 +248,7 @@ clear_feedback_filter = "Clear Filter"
 clear_selection = "Clear Selection (%%count%%)"
 clear_tag_filter = "Clear Filter"
 close = "close"
+Close broadcast for this session = "Close broadcast for this session"
```

Comment:
Is this string used anywhere? I did a search and couldn't find where it was used -- I was trying to better understand its context.